examples/src/chains/sql_db_sql_output.ts

```diff
@@ -0,0 +1,35 @@
+import { DataSource } from "typeorm";
+import { OpenAI } from "langchain/llms/openai";
+import { SqlDatabase } from "langchain/sql_db";
+import { SqlDatabaseChain } from "langchain/chains";
+
+/**
+ * This example uses Chinook database, which is a sample database available for SQL Server, Oracle, MySQL, etc.
+ * To set it up follow the instructions on https://database.guide/2-sample-databases-sqlite/, placing the .db file
+ * in the examples folder.
+ */
+export const run = async () => {
```

Comment:
Btw we can stop using the run wrapper function in examples, it works without it now